src/main/java/teammates/ui/template/InstructorStudentListFilterBox.java

```diff
@@ -0,0 +1,24 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+public class InstructorStudentListFilterBox {
+
+    private List<InstructorStudentListFilterCourse> courses;
+    private boolean displayArchive;
```

Comment:
Will this be used on multiple pages with the same attributes required?
I'm not quite sure if it's worth abstracting this into its own class instead of simply leaving it as a List and boolean on the `*PageData`.